src/plugins/intel_cpu/src/nodes/fullyconnected.cpp

```diff
@@ -78,9 +78,9 @@ ov::element::TypeVector FullyConnected::getSupportedCompressedActivationsTypes()
 bool FullyConnected::isSupportedOperation(const std::shared_ptr<const ov::Node>& op,
                                           std::string& errorMessage) noexcept {
     try {
-        if (!ov::is_type<const ov::op::internal::FullyConnected>(op) &&
-            !ov::is_type<const ov::op::internal::FullyConnectedQuantizedLegacy>(op) &&
-            !ov::is_type<const ov::op::internal::FullyConnectedCompressed>(op)) {
+        if (!ov::is_type_any_of<const ov::op::internal::FullyConnected,
+                                const ov::op::internal::FullyConnectedQuantizedLegacy,
+                                const ov::op::internal::FullyConnectedCompressed>(op)) {
```

Comment:
```suggestion
        if (!ov::is_type<const ov::op::internal::FullyConnected>(op)) {
```
As `FullyConnected` is base of all these classes, check others similar case maybe many checks is not required at all.